hls-plugin-api/src/Ide/Plugin/ConfigUtils.hs

```diff
@@ -106,7 +106,7 @@ pluginsToVSCodeExtensionSchema IdePlugins {..} = A.object $ mconcat $ singlePlug
         genericSchema =
           let x =
                 [toKey' "diagnosticsOn" A..= schemaEntry "diagnostics" | configHasDiagnostics]
-                  <> nub (mconcat (handlersToGenericSchema <$> handlers))
+                  <> Data.List.Extra.nubOrd (mconcat (handlersToGenericSchema <$> handlers))
```

Comment:
:information_source: Because of rule defined here

https://github.com/andys8/haskell-language-server/blob/ebb58eec6fb9817c163879800cf0c17dcb25e460/.hlint.yaml#L47